gateway/src/apicast/policy_chain.lua

```diff
@@ -87,7 +88,12 @@ function _M.load_policy(module, version, ...)
         local mod, err = policy_loader:pcall(module, version or 'builtin')
 
         if mod then
-            return mod.new(...)
+            local new_policy_ok, res = pcall(mod.new, ...)
```

Comment:
It might be good to expect a second return value `err`. It is possible a policy could `return nil, 'error'`.

Reply:
Fixed :+1: